beets/importer.py

```diff
@@ -222,19 +222,31 @@ def set_config(self, config):
             iconfig['resume'] = False
             iconfig['incremental'] = False
 
-        # Copy, move, link, and hardlink are mutually exclusive.
+        if iconfig['reflink']:
+            iconfig['reflink'] = iconfig['reflink'] \
+                .as_choice(['auto', True, False])
```

Comment:
This looks a little odd—unless I'm mistaken, `iconfig` is just a flat dictionary rather than a Confuse view, right? So I'm not sure `.as_choice` is supported on it. (Maybe I'm missing something obvious?)

Reply:
I'm quite unsure here, I think I copied that from a few lines below, but then, this patch is kind of old. Three years or so :joy:

I don't really recall how this config method is supposed to work. FWIW, I've had
```yaml
import:
  reflink: auto
```
in my laptop's beets config on this branch for the last year or two, but then this config is also a transcoding setup, where reflink doesn't make sense at all.